libs/verifiable-credentials/src/main/java/uk/gov/di/ipv/core/library/verifiablecredential/domain/VerifiableCredentialStatus.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.di.ipv.core.library.verifiablecredential.domain;
+
+import lombok.Getter;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Getter
+public enum VerifiableCredentialStatus {
+    CREATED("created"),
+    PENDING("pending");
+    private static final Map<String, VerifiableCredentialStatus> STATUSES =
```

Comment:
Will there be a state when this is approved or one where it's rejected?

Reply:
Not to my knowledge - the ADR only indicates pending and created responses. I suspect you may well be correct going forward.